web/modules/custom/dpl_instant_loan/src/Form/DplInstantLoanSettingsForm.php

```diff
@@ -84,15 +84,13 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#default_value' => $config->get('enabled'),
     ];
 
-    $form['match_string'] = [
-      '#type' => 'textfield',
-      '#title' => $this->t('Match String', [], ['context' => 'dpl_instant_loan']),
-      '#description' => $this->t(
-        'Text used to identify materials which are available for instant loans. This text must be present in the material group of such materials.',
-        [],
-        ['context' => 'dpl_instant_loan']
-      ),
-      '#default_value' => $config->get('match_string'),
+    $form['match_strings'] = [
+      '#type' => 'textarea',
+      '#title' => $this->t('Match Strings', [], ['context' => 'dpl_instant_loan']),
+    // Set the number of visible rows for the textarea.
```

Comment:
```suggestion
      // Set the number of visible rows for the textarea.
```

Indentation seems off here.